index.html

```diff
@@ -5,8 +5,91 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="styles/index.css" rel ="stylesheet">
 </head>
 <body>
-    
+    <div class ='header' id ='pageTitle'>
+        <h1>Weather Report</h1>
+        <div class ='header' id='lovelyCity'>
+            <p>for the lovely city of</p>
+        </div>
+        <div class ='header' id='mutableSubHeader'>
+        </div>
+    </div>
+    <main class ='websiteBody'>
+        <section class ='mutableComponents'>
+            <div class = 'widget' id ='temperatureChanger'>
+                <div class = 'widget text' id ='Temperature Title'>
+                    <p>Temperature</p>
+                </div>
```

Comment:
It's more appropriate to use a heading tag (`<h2>`, `<h3>` depending on how nested) for something like a `<section>` title:

```suggestion
                <h2 class = 'widget text' id ='Temperature Title'>
                    Temperature
                </h2>
```